### PR TITLE
Enable IDE0060

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/Logging.AzureAppServices/src/BatchingLogger.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public bool IsEnabled(LogLevel logLevel)
         return _provider.IsEnabled;
     }
 
-    public void Log<TState>(DateTimeOffset timestamp, LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+    public void Log<TState>(DateTimeOffset timestamp, LogLevel logLevel, EventId _, TState state, Exception exception, Func<TState, Exception, string> formatter)
```
Comment (Tratcher) [inline src/Logging.AzureAppServices/src/BatchingLogger.cs:31]: Shouldn't the EventId get logged?

File: src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs [hunk 2]
```diff
@@ -869,18 +869,16 @@ private async Task InvokeStreamCore(ConnectionState connectionState, string meth
...
+    private async Task SendHubMessage(ConnectionState connectionState, HubMessage hubMessage)
     {
         _state.AssertConnectionValid();
         _protocol.WriteMessage(hubMessage, connectionState.Connection.Transport.Output);
 
         Log.SendingMessage(_logger, hubMessage);
 
-#pragma warning disable CA2016 // Forward the 'CancellationToken' parameter to methods
         // REVIEW: If a token is passed in and is canceled during FlushAsync it seems to break .Complete()...
         await connectionState.Connection.Transport.Output.FlushAsync().ConfigureAwait(false);
```
Comment (Tratcher) [inline src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs:880]: @BrennanConroy @halter73 ?

**Review by Tratcher — APPROVED** [review]

File: src/Logging.AzureAppServices/src/BatchingLogger.cs (hunk 1 above)
Comment (pranavkm) [reply src/Logging.AzureAppServices/src/BatchingLogger.cs:31]: This method turns the log to a string. We could add it to the log, but I'm not super familiar who the consumer is and if changing the format would affect it in any way.

File: src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs (hunk 2 above)
Comment (BrennanConroy) [reply src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs:880]: Let's not make this change.

File: src/Servers/IIS/IIS/src/Core/OutputProducer.cs [hunk 3]
```diff
@@ -51,7 +50,7 @@ public void Complete()
         }
     }
 
-    public void Abort(Exception error)
```
Comment (BrennanConroy) [inline src/Servers/IIS/IIS/src/Core/OutputProducer.cs:54]: Should the pipe be completed with this exception?

File: src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs (hunk 2 above)
Comment (pranavkm) [reply src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs:880]: Could you clarify - should I suppress the warning or is this change that stops passing the token fine?

Comment (BrennanConroy) [reply src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs:880]: Revert the change, we obviously want to pass the token just haven't gotten around to figuring out the issue.

File: src/Servers/IIS/IIS/src/Core/OutputProducer.cs (hunk 3 above)
Comment (pranavkm) [reply src/Servers/IIS/IIS/src/Core/OutputProducer.cs:54]:
```diff
- _pipe.Writer.Complete();
+ _pipe.Writer.Complete(error);
```

Like so?

Comment (BrennanConroy) [reply src/Servers/IIS/IIS/src/Core/OutputProducer.cs:54]: Yeah, would have to look at the usage to see if we want that though.

**Pull request merged**